app/assets/javascripts/initializers/initializePaymentPointers.js

```diff
@@ -2,10 +2,10 @@ function initializePaymentPointers() {
   var userPointer = document.getElementById('author-payment-pointer');
   var basePointer = document.getElementById('base-payment-pointer');
   var meta = document.querySelector("meta[name='monetization']");
-  
+
   if (userPointer) {
     meta.content = userPointer.dataset.paymentPointer;
-  } else {
+  } else if (basePointer) {
```

Comment:
basePointer will not exist if the site config payment pointer is not configured, hence we put it in an else if